cpp/log/cert.cc

```diff
@@ -1308,7 +1341,10 @@ Cert::Status PreCertChain::UsesPrecertSigningCertificate() const {
     return Cert::FALSE;
   }
 
-  return issuer->HasExtendedKeyUsage(cert_trans::NID_ctPrecertificateSigning);
+  StatusOr<bool> issuer_eku_status =
```

Comment:
Can drop the `_status` from the name here, or, even, eliminate the variable (just pass it straight in)?